### DISCUSSION TITLE
Using the new GitHub Issues

### DISCUSSION THREAD

**Discussion by andersy005** [ProjectPythia/projectpythia.github.io#176] [Ideas]
A while ago I signed up for the new **[GitHub Issues](https://github.com/features/issues/) (still beta)**. Today, I received a notification that the `ProjectPythia` account is enrolled in the public beta:

![Screen Shot 2021-10-15 at 3 27 47 PM](https://user-images.githubusercontent.com/13301940/137555469-53277ecd-e05c-473a-84f6-591b5dab624d.png)

Seems it would be worth trying this out for managing and tracking issues and pull requests, etc across multiple repositories. 

- To access this new/beta feature, head over to  https://github.com/orgs/ProjectPythia/projects?type=beta
- The docs reside here: https://docs.github.com/en/issues/trying-out-the-new-projects-experience
- Here's a video with more details
[![Plan and track projects closer to your code](https://img.youtube.com/vi/SI1ra-XHWHM/0.jpg)](https://www.youtube.com/watch?v=SI1ra-XHWHM "Plan and track projects closer to your code")

**Comment by brian-rose** [comment]
Trying this out could be an activity for the next Pythia hackathon on 10/28
